index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Project</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+   <header>
+     <div class="welcome">
+        <h1>Hi there</h1>
+     </div>
```

Comment:
- [ ] Your logo should be wrapped in an `a` tag. Please replace this `h1` tag with the `a` tag. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.